WordPress/src/main/res/layout/post_list_activity.xml

```diff
@@ -23,19 +23,31 @@
             app:theme="@style/ThemeOverlay.AppCompat.Dark.ActionBar">
         </android.support.v7.widget.Toolbar>
 
-        <android.support.design.widget.TabLayout
-            android:id="@+id/tabLayout"
+        <LinearLayout
             android:layout_width="match_parent"
-            android:layout_height="@dimen/tab_height"
-            android:layout_gravity="start"
-            android:background="@color/color_primary"
-            app:tabContentStart="@dimen/toolbar_content_offset"
-            app:tabGravity="fill"
-            app:tabIndicatorColor="@color/white"
-            app:tabMode="scrollable"
-            app:tabSelectedTextColor="@color/white"
-            app:tabTextColor="@color/white"
-            app:theme="@style/Base.Widget.Design.TabLayout"/>
+            android:layout_height="wrap_content"
+            android:orientation="horizontal">
+
+            <android.support.v7.widget.AppCompatSpinner
+                android:id="@+id/post_list_author_selection"
+                style="@style/Widget.AppCompat.Button.Borderless"
+                android:layout_width="@dimen/posts_list_spinner_width"
+                android:layout_height="match_parent"
+                android:padding="0dp"/>
```

Comment:
Do we need to set the padding to `0dp` manually?

Reply:
I can double check

Reply:
Looks like this is indeed required since the `Widget.AppCompat.Button.Borderless` style has padding set which clips our icon area if it isn't set back to `0dp`

Reply:
Thank you for checking! 🙇